genconvert/src/main/java/com/google/myanmartools/CompileTranslit.java

```diff
@@ -0,0 +1,463 @@
+/* Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.myanmartools;
+
+import com.ibm.icu.text.Transliterator;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+/** A Java binary to generate Javascript and Java transliterators from an ICU4J transliteration file. The
+ * output is a) Javascript code, and b) Java code that take text and perform the transliterations specified.
+ *
+ * <p>Note: this does not implement many features of ICU Transliteration, but does support
+ * conversions such as Zawgyi to Unicode for Burmese language.
+ */
+
+public final class CompileTranslit {
```

Comment:
Since this is a utility class, not public API, I'm not going to be too picky with my comments.